domains/QWERTY.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "_QWRTY_QAZ"
+    },
+    "record": {
+        "CNAME": "https://pirate.elastic-gbm.com/"
```

Comment:
CNAME records must be hostnames without any other protocols or file paths. See docs for more information: https://docs.is-a.dev